packages/cli/lib/errorHandlers/overrrideErrors.js

```diff
@@ -0,0 +1,95 @@
+const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/apiErrors');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+
+const { PLATFORM_VERSION_ERROR_TYPES } = require('../constants');
+const { i18n } = require('../lang');
+const { uiLink } = require('../ui');
+
+const i18nKey = 'cli.lib.errorHandlers.overrideErrors';
+
+function createDashes() {
+  return '-'.repeat(79);
+}
+
+function createPlatformVersionError(subCategory, errorData) {
+  const platformVersion = errorData.context.PLATFORM_VERSION[0];
```

Comment:
@ypzheng, will there only ever be one item in the `PLATFORM_VERSION` array? Could this simply be a string? It feels brittle to me right now.

Reply:
Did look into this a bit. It seems like there is no easy way to make it a string due to how errors are transformed and parsed along backend clients & services.

However, there will only be one string in `PLATFORM_VERSION` because the way we construct platform version related error on backend is always matching one single string to an error token (i.e. PLATFORM_VERSION_RETIRED). 

I'll keep this in mind though. If I find anything better in the future.